_rules/html-page-lang-xml-lang-match-5b7ae0.md

```diff
@@ -83,7 +83,7 @@ Since most assistive technologies will consistently use `lang` over `xml:lang` w
 
 #### Failed Example 1
 
-`html` element with non-matching primary language subtags for `lang` and `xml:lang`.
+This `html` element has different [primary language subtags][] for its `lang` and `xml:lang` attributes.
 
 ```html
 <html lang="fr" xml:lang="en"></html>
```

Comment:
Should there be a second failing example where the primary subtags do not match but the extended subtags do not? 

Appreciate the PR is about the language being used in the rule, rather than the rule itself.

Reply:
Could be a good idea. Not sure. Totally subject for another PR in any cases 😄